src/pages/Hwangsoohyeon/Join/Join.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import './Join.scss';
+import { useNavigate } from 'react-router-dom';
+
+const Join = () => {
+  const navigate = useNavigate();
+  const [inputid, setinputid] = useState('');
+  const [inputpw, setinputpw] = useState('');
+
+  const goToJoin_ok = () => {
+    navigate('/Join_ok');
+  };
+
+  const goToLogin = () => {
+    navigate('/Login');
+  };
+
+  const saveUserId = (e) => {
+    setinputid(e.target.value);
+    console.log(inputid);
+  };
+
+  const saveUserPw = (e) => {
+    setinputpw(e.target.value);
+    console.log(inputpw);
+  };
+
+  const able = inputid.indexOf('@', '.') !== -1 && inputpw.length >= 10;
+
+  return (
+    <div>
+      <div className="Back">
+        <div className="Backicon">
+          <img
+            className="Back_arrow"
+            src="./images/Back_arrow.png"
+            alt="Back_arrow"
+            onClick={goToLogin}
+          />
+        </div>
+        <header>뒤로</header>
+      </div>
+      <div className="Join1">
+        <div className="info">
+          {/* <div>회원가입페이지</div> */}
+          <h1>회원가입</h1>
+
+          <div className="info1">
+            <div className="info_label">
+              <h2>기본정보</h2>
+              <label>필수사항</label>
+            </div>
+            <input
+              type="text"
+              placeholder="이메일"
+              onChange={saveUserId}
```

Comment:
혹은 onChange에 연결된 실행함수에 e.target.value를 직접 넘겨주는 방식도 생각해볼 수 있습니다
```suggestion
              onChange={(e)=>saveUserId(e.target.value)}
```